MIP/mip-34/README.md

```diff
@@ -0,0 +1,261 @@
+
+
+![Version A Diagram](postconfirmationV1.png)
+*Figure 1: Leader-dependent L2-block generation process in Version A.*
+
+#### Version B: Deterministic blocks
+
+Blocks are deterministically derived from the sequencer-batch, and consequently the block-range $B_r'$ is deterministic. This is in contrast to Version A, where a leader proposes the next transition.Validators then attest for the next transition directly:  $B_r \xrightarrow{\ txs \ } B_r'$. E.g. by commiting to the the hash of $B_r'$.
+
+An additional actor - the `acceptor` - is introduced that initiates the postconfirmation process. This is necessary, as this step requires additional gas costs on L1 and thus this role requires additional rewards. The `acceptor` serves for a specified period and is then replaced by another validator. 
+
+> [!NOTE]
+> Since the block derivation is deterministic, $f+1$ may be sufficient to confirm the block. (However, we require $2f+1$ to cover potential edge cases, such as that the sequencer cannot be trusted.)
+
+**Direct L1 commitments**. In the scenario where validators commit individually they send the block hashes of the calculated blocks directly to the L1 contract. 
+
+**Acceptor collects commitments**. In a more optimised scenario, the acceptor sends the super-majority proof to the L1 contract. 
+
+![Version A Diagram](postconfirmationV2.png)
+*Figure 2: Leader-independent (deterministic) L2-block generation process in Version B.*
+
+
+#### Version C: Deterministic blocks and L2-finality certificates
+
+This approach extends Version B. 
+
+A p2p layer is established between validators. Validators communicate to aggregate a threshold of votes on each deterministically determined L2-block. This provides L2-confirmations in the order of seconds.
+
+Since this approach already collects commitments off-L1, the natural choice is to use Version B where the acceptor collects the commitments and sends the super-majority proof to the L1 contract. This provides postconfirmations in the order of minutes.
+
+
+![Version A Diagram](fullDesign.png)
+*Figure 3: Leader-independent (deterministic) L2-block generation process in Version C. Validators co-operate to create a L2-finality certificate before L1 is involved.*
+
+## Verification
+
+### Correctness and Security
+
+A more detailed discussion on the correctness and security is discussed in [this blog post on FFS](https://blog.movementlabs.xyz/article/security-and-fast-finality-settlement).
+
+The level of security depends on the total stake of the L2 validators. The higher the more secure.
+
+The correctness of the mechanism relies on a few trust assumptions.
+
+**Byzantine assumption**. 
+We assume that at most $f$ of the total $n=3f+1$ (L2) validators can be malicious.
```

Comment:
There's a bit more to this I would like to reference in a definition of consensus. But, I will hold off here...

Reply:
@l-monninger But this is not a consensus, so we don't need the definition of consensus.